package.json

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "cli",
+	"version": "0.0.0",
+	"license": "MIT",
+	"bin": {
+		"asyncapi": "dist/cli.js",
+		"@asyncapi/cli": "dist/cli.js"
+	},
+	"engines": {
+		"node": ">=10"
+	},
+	"scripts": {
+		"build": "tsc",
+		"start": "npm run build && node dist/cli.js",
+		"pretest": "npm run build",
+		"pretest:coverage": "npm run build",
+		"test": "jest",
+		"test:coverage": "jest --verbose --coverage"
+	},
+	"files": [
+		"dist/cli.js"
+	],
+	"dependencies": {
+		"@asyncapi/parser": "^1.4.5",
+		"commander": "^7.1.0",
```

Comment:
I don't think it is used

Reply:
@jotamusik please remove commander if not used and update package-lock

Reply:
done!